File: Java/백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int n = Integer.parseInt(br.readLine());
		StringTokenizer st = new StringTokenizer(br.readLine());
		int max = -1000000, min = 1000000;
		for(int i = 0; i < n; i ++) {
			int element = Integer.parseInt(st.nextToken());
			if(max < element) max = element;
			if(min > element) min = element;
		}
		System.out.println(min + " " + max);
	}
}

File: Kotlin (JVM)/백준/Bronze/10818. 최소， 최대/최소， 최대.kt
import java.util.*

fun main(args: Array<String>){
    var N: Int = readLine()!!.toInt()
    var scan = Scanner(System.`in`)
    var arr: IntArray = IntArray(N, {0})
    for (i: Int in 0 until N)
        arr[i] = scan.nextInt()

    var max: Int = arr[0]
    var min: Int = arr[0]

    for(i: Int in 1 until N){
        if(max < arr[i]) max = arr[i];
        if(min > arr[i]) min = arr[i];
    }

    println("$min $max")
}